manuscript/revision_response.Rmd

```diff
@@ -0,0 +1,113 @@
+
+For this question, it’s best to take a look at Figure 2 to see how closely the RL policy approximates the optimal policy for the fishing example. The agent seems to learn the general functional form of the policy but is slightly biased. We will address the re-occurring concern of whether RL is ready to be used for real problems in another comment.
+
+5) While preventing tipping points is a major goal of modern interdisciplinary science, conservation policy using RL only in one instance is not enough to comment on its applicability. The authors should validate the method by testing on a model of different origins with a different non-linearity and available real data exhibiting a critical transition. (As saddle-node bifurcation to all such instances, the trained RL model should work well when tested on such data). This shall be instrumental in commenting on the generality of the approach for conserving tippings. 
+The authors should justify the robustness of their RL method at conserving tipping in real empirical data. While such data may not be available, noisy simulated data with imperfect sampling mimicking real scenarios can be considered. Real data are often irregularly sampled with errors in reported values (quite obvious in climate data and maybe a concern while reducing emissions to prevent climate tippings ). The authors may perform an experiment and discuss the details in the discussion section as to how RL will deal with such instances.
+
+We share this perspective with the reviewer that our study does not prove that deep RL can be immediately applied to conservation problems. Validating RL as method for ecological problems is not the intention of our paper. Instead, we have written our paper to be an introduction for ecologists to deep RL, whereby we introduce the OpenAI Gym framework, some fundamental RL concepts and example problems, so that researchers can investigate deep RL on their own and better understand how it is beginning to be applied to ecology. We are at a point in time where researchers and organizations are using deep RL in conservation contexts, so it is imperative for our community to start understanding the benefits and downfalls of using deep RL for our specialized class of problems.
+
+!!edit intro to to voice this intention CC Carl!!
+
+6) In Appendix C, the authors mentioned that scaling the instance space to [-1 1] improves the agent performance of RL. Ecological events are highly non-linear sequences of events; such data are often non-stationary. Will scaling the state space in RL not incorporate bias and still lead to robust predictions. I think it demands clarifications.
+
+Re-scaling the state space is equivalent to measuring the process in different units, so this does not affect the transition dynamics.
```

Comment:
I think we should add this to appendix C.  Here we can just say: "we have added a discussion of scaling in the appendix".  We might note that this does assume some appropriate knowledge of the 'natural units' for non-dimensionalization (i.e. rescale by carrying capacity).  We can point to Alan's book which has a nice discussion of non-dimensionalization for ecologists: https://link.springer.com/book/10.1007/978-1-4757-2731-9